lambda_functions/va_profile/va_profile_opt_in_out_lambda.py

```diff
@@ -6,9 +6,15 @@
 import os
 import psycopg2
 import sys
+from http.client import ConnectionError, HTTPSConnection
+from json import dumps
+
 
 OPT_IN_OUT_QUERY = """SELECT va_profile_opt_in_out(%s, %s, %s, %s, %s);"""
 NOTIFICATION_API_DB_URI = os.getenv("notification_api_db_uri")
+VA_PROFILE_DOMAIN = "https://api.va.gov"
```

Comment:
This isn't in the VA Profile sync document, and I'm not sure that it is correct.

Reply:
I agree.  Was there a domain specified in the docs?  I know we used a route and defined the method.

Reply:
It's not in the doc.  I e-mailed Josh L. and got the answer I need, but now I need to update the infra code with some new variables.